### PR TITLE
[VAN-2024] Initial PR for Vancouver 2024

### PR DESCRIPTION
**Pull request by terraboops**
I am setting up the event website for Vancouver 2024, attempting to rekindle the event with a lot of new folks.

I will email info@devopsdays.org with the complete list of organizers with emails.

Thanks!

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -117,8 +117,13 @@
 /texas/*		/events/2021-texas/:splat		302
 /tokyo/*		/events/2022-tokyo/:splat		302
 /toronto/*		/events/2020-toronto/:splat		302
+<<<<<<< Updated upstream
```
Comment (glasnt) [inline static/_redirects:120]: Looks like you have some git rebase fluff left here, causing CI errors.

Comment (terraboops) [reply static/_redirects]: Oops! Sorry about that. Fixed!

**Comment by glasnt** [comment]
Did you copy your 2021 event to create this PR, or use the `utilities/add_new_event.sh` script? We ask that events use the script to ensure you aren't copying across old items, and are using a fresh instance each time. 

If this PR was copied from last time, please close this one and use the script to create a new PR. Thanks!

**Comment by terraboops** [comment]
@glasnt I used the script :)

Just to be sure, I re-ran the script and the git diff was clear. Thanks for checking!

**Pull request closed without merging**